test/gst-msdk/encode/vp9.py

```diff
@@ -0,0 +1,93 @@
+###
+### Copyright (C) 2018-2019 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ..util import *
+from .encoder import EncoderTest
+
+spec = load_test_spec("vp9", "encode", "8bit")
+
+class VP9EncoderTest(EncoderTest):
+  def before(self):
+    vars(self).update(
+      codec         = "vp9",
+      gstencoder    = "msdkvp9enc",
+      gstdecoder    = "ivfparse ! vaapivp9dec",
+      profile       = "profile0"
+    )
+    super(VP9EncoderTest, self).before()
+
+  def get_file_ext(self):
+    return "ivf"
+
+class cqp_lp(VP9EncoderTest):
+  @platform_tags(VP9_ENCODE_8BIT_LP_PLATFORMS)
+  @slash.requires(*have_gst_element("msdkvp9enc"))
+  @slash.requires(*have_gst_element("msdkvp9dec"))
+  @slash.parametrize(*gen_vp9_cqp_lp_parameters(spec))
+  def test(self, case, ipmode, qp, quality, slices, refmode, looplvl, loopshp):
+    slash.logger.notice("NOTICE: 'refmode' parameter unused (not supported by plugin)")
+    slash.logger.notice("NOTICE: 'looplvl' parameter unused (not supported by plugin)")
+    slash.logger.notice("NOTICE: 'loopshp' parameter unused (not supported by plugin)")
+    vars(self).update(spec[case].copy())
+    vars(self).update(
+      case      = case,
+      gop       = 30 if ipmode != 0 else 1,
```

Comment:
What about `lowpower = True`?

Reply:
according to gst-msdk, it only support vp9 lp mode and it has no para "low-power" in vp9enc, so it does not need this variable.